File: corpus/2017q1/_p2q1_ok/_check/D;0074.java

import java.util.Scanner;

public class BrunaRibeiroOliveira_EXERCICIO 1  {

    public static void main(String[]args){

    int A[] = {-62,8,98,75,26,77};
    int B[] = {-102,0,102,0,26,79};
    int C[] = new int [6];

    ImprimeVetor (A);
    ImprimeVetor (B);

    C = MaioresElementos (A,B);
    ImprimeVetor (C);

    }

    public static void ImprimeVetor (int Vet1[]){
    int A[] =  {-62,8,98,75,26,77};
    int B[] = {-102,0,102,0,26,79};
    Scanner input = new Scanner (System.in);

        for (int i=0; i<6; i++){
            System.out.print(A[i]+"   ");
        }
        System.out.println("");        
        for (int j=0; j<6; j++){
            System.out.print(B[j]+"   ");
        }
        System.out.println("");
    }

    public static int MaioresElementos (int Vet1[], int Vet2[]){
    int A[] =  {-62,8,98,75,26,77};
    int B[] = {-102,0,102,0,26,79};
    int C[] = new int [6];
    int x = 0; 
    int xmaior = 0;

    Scanner input = new Scanner (System.in);

        for (int i=0; i<6; i++){
            if (A[i]>x){
                x = xmaior;                

            }
        }
    }

}

File: corpus/2017q1/_p2q1_ok/_check/D;0074.py
A = [-62,8,98,75,26,77]
B = [-102,0,102,0,26,79]
ImprimeVetor (A)
ImprimeVetor (B)
C = MaioresElementos (A,B)
ImprimeVetor (C)
def ImprimeVetor(int Vet1[]):
    A = [-62,8,98,75,26,77]
    B = [-102,0,102,0,26,79]
    Scanner input = new Scanner (System.in)
    for i=0 in range(0, 6):
        print(A[i]+"   ")
    print("")
    for j=0 in range(0, 6):
        print(B[j]+"   ")
    print("")
def MaioresElementos(int Vet1[], int Vet2[]):
    A = [-62,8,98,75,26,77]
    B = [-102,0,102,0,26,79]
    int x = 0
    int xmaior = 0
    Scanner input = new Scanner (System.in)
    for i=0 in range(0, 6):
        if A[i]>x:
            x = xmaior